### DISCUSSION TITLE
Add the ability to use issue forms in the mobile app

### DISCUSSION THREAD

**Discussion by SuperFola** [community/community#20790] [Mobile]
Currently, we can fill in issues, check PRs status and even check status (which redirected to a browser before, now this is nearly all in-app and it is great).

However when issue forms are enabled (eh on https://github.com/ArkScript-lang/Ark), trying to fill a new issue in the mobile app will redirect to a browser instead of staying in the app. I guess this hasn't been implemented yet, but I feel like it will be needed in the future, as the new issue type are gradually becoming available to more people.

**Comment by iskae** [comment]
Hi @SuperFola 👋 

Thanks a bunch for reaching out and your feedback 🙇  It is extremely crucial for us to receive feedback from our users, so please keep them coming ❤️ 

As you have mentioned we are currently only supporting issue templates and not the issue forms. We currently do not have any plans for it but we have an internal tracking issue for it and I will be adding your request there too 👍 Please stay tuned for our future releases 😊

**Reply by SuperFola** [reply]
Thanks!
